bigbluebutton-html5/imports/startup/client/intl.jsx

```diff
@@ -136,6 +137,8 @@ class IntlStartup extends Component {
               }
 
               const dasherizedLocale = normalizedLocale.replace('_', '-');
+              const { language, formattedLocale } = formatLocaleCode(dasherizedLocale);
+
               this.setState({ messages: mergedMessages, fetching: false, normalizedLocale: dasherizedLocale }, () => {
```

Comment:
```suggestion
              this.setState({ messages: mergedMessages, fetching: false, normalizedLocale: formattedLocale }, () => {
```

I think you can set it right away and access it later in `render()`.